returnn/datasets/basic.py

```diff
@@ -799,6 +801,7 @@ def get_data(self, seq_idx, key) -> numpy.ndarray:
         :param str key: data-key, e.g. "data" or "classes"
         :return: features or targets: format 2d (time,feature) (float)
         """
+        self._last_cur_seq_idx = max(self._last_cur_seq_idx, seq_idx)
```

Comment:
Why do you need that?

Reply:
I'm still wondering how to map the indices between their respective spaces. Keeping the index inside the dataset circumvents any mapping. But I also wrote this in the other comment.